src/app/shared/axiosHelper.ts

```diff
@@ -34,37 +33,70 @@ const extractCredentialIssuerId = (
   return undefined;
 };
 
+const sanitiseResponseData = (response: AxiosResponse): object | undefined => {
+  try {
+    if (typeof response.data === "object") {
+      const body = { ...response.data };
+
+      const endpoint = response.config?.url ?? "";
+      if (endpoint.includes("proven-identity-details")) {
```

Comment:
Could we make this an allow list rather than deny list ie. specify which requests we should allow data logging for (which I think should be all except proven-identity-details)? so we don't inadvertently start leaking PII if we add new requests in the future

Also could we pull the values from config eg. `API_SESSION_INITIALISE` rather than hardcoding?